src/ape/utils.py

```diff
@@ -194,13 +195,44 @@ def extract_nested_value(root: Mapping, *args: str) -> Optional[Dict]:
     return current_value
 
 
+class GitHubClient:
```

Comment:
I am using this class on another branch to also download packages from GitHub

Reply:
nit to match
```suggestion
class GithubClient:
```